geometry_msgs/Pose2D pose1
geometry_msgs/Pose2D pose2
geometry_msgs/Pose2D vel
float32 height
